frigate/embeddings/onnx/lpr_embedding.py

```diff
@@ -0,0 +1,297 @@
+import logging
+import os
+import warnings
+
+import cv2
+import numpy as np
+
+from frigate.comms.inter_process import InterProcessRequestor
+from frigate.const import MODEL_CACHE_DIR
+from frigate.types import ModelStatusTypesEnum
+from frigate.util.downloader import ModelDownloader
+
+from .base_embedding import BaseEmbedding
+from .runner import ONNXModelRunner
+
+warnings.filterwarnings(
+    "ignore",
+    category=FutureWarning,
+    message="The class CLIPFeatureExtractor is deprecated",
+)
+
+logger = logging.getLogger(__name__)
+
+LPR_EMBEDDING_SIZE = 256
+
+
+class PlateDetectionEmbedding(BaseEmbedding):
+    def __init__(
+        self,
+        model_size: str,
+        requestor: InterProcessRequestor,
+        device: str = "AUTO",
+    ):
+        super().__init__(
```

Comment:
In all these calls to `super().__init__` in this refactor, it might be good to just name the parameters for readability, eg:

```
super().__init__(
    model_name="jinaai/jina-clip-v1",
    model_file="...",
    download_urls={
        model_file: f"https://huggingface.co/jinaai/jina-clip-v1/resolve/main/onnx/{model_file}",
        "preprocessor_config.json": "https://huggingface.co/jinaai/jina-clip-v1/resolve/main/preprocessor_config.json",
    },
)
```